kmath-nd4j/src/main/kotlin/space/kscience/kmath/nd4j/arrays.kt

```diff
@@ -8,3 +8,13 @@ package space.kscience.kmath.nd4j
 import space.kscience.kmath.misc.toIntExact
 
 internal fun LongArray.toIntArray(): IntArray = IntArray(size) { this[it].toIntExact() }
+
+internal fun LongArray.linspace(start: Long, stop: Long) = Array(this.size) {
```

Comment:
We try not to use Arrays in mathematics because they have some default overrides in Kotling StdLib. You should use Buffer and there are already BufferAlgebra.zero producers for that. There is a DoubleBuffer producer that uses step, we can add a similar one for Longs, but we need to be careful about naming.